guardtime/js-common
# Update dependencies and package-lock to v2

* Update package.json to version 2
* Add pkcs7 verification
* Add file extension check to linter
* Update datahasher to use new library instead of node-forge
* Remove node-forge library
